### ISSUE TITLE
Pro Menu dropdowns off screen.

### ISSUE THREAD

**Issue by shoelaced** [soflyy/oxygen-bugs-and-features#1286]
**Describe the bug**
*A clear and concise description of what the bug is.*

If the pro menu is used in a sidebar with `position: fixed` and the dropdowns contain more items than will fit on-screen, the bottom of the dropdown gets cut off.

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
*Yes/No*

Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
*If yes, provide the link to a Sandbox install where the issue is present. If the issue only exists on a specific post or template, provide a direct link to that post or template.*

I have not tried yet, wanted to report before I forget.

**To Reproduce**

1. Create a fixed sidebar.
2. Add a Pro Menu.
3. Add a page to the menu.
4. Add 30 pages or so - however many it takes for the dropdown to be too long.
5. Note that it goes below the bottom of the viewport and does not trigger the scrollbars, making the lower menu items unavailable.

**Expected behavior**
*A clear and concise description of what you expected to happen.*

Ideally it would do what the Pro Menu already does for the right side of the screen: Move the dropdowns so that they never display off-screen. If the menu should happen to get so tall that it is too tall for the viewport, it should trigger the scrollbar. But before that, it should move itself up as far as needed to prevent it flowing out of the viewport.

**What actually happens**
*A clear and concise description of what happens instead of the expected behavior.*

As described above, the menu dropdowns go below the viewport, rendering them useless.

**Screenshots**
*If applicable, add screenshots to help explain your problem.*

In this screenshot, my mouse is hovering over the lowest available menu item, but there are three off-screen and no way to get to them:

![image](https://user-images.githubusercontent.com/8772661/85962671-d4431880-b9a9-11ea-939e-e9ff27d8bcaa.png)

**Comment by KittenCodes** [comment]
@shoelaced 

Does adding overflow: auto to the fixed sidebar allow the scrollbars to show when needed?

**Comment by shoelaced** [comment]
Yes, but then this happens:

![image](https://user-images.githubusercontent.com/8772661/86134435-85989a00-bae1-11ea-8257-15e57a06963f.png)

**Comment by shoelaced** [comment]
(and setting `overflow-y: auto; overflow-x: visible` does not work for some reason)

**Comment by KittenCodes** [comment]
@shoelaced Please could you replicate this on the page here: https://oxygen-qm45fwrcr799k.oxygen-demo.qsandbox.me/wp-admin/post.php?post=98&action=edit ? I've added a large menu to WordPress.

The username and password for the Sandbox site are both demo.

**Comment by KittenCodes** [comment]
Thanks @shoelaced. What browser and operating system are you using? If I load the page in Chrome on Windows 10, the page does scroll to allow access to the off-screen menu items.

However, I have noticed that the page length is automatically increased to allow for the long menu, even when it's not visible.

**Comment by shoelaced** [comment]
Hi @KittenCodes,

I'm in Chrome (latest) in Windows 10, but on my end it does not scroll. Instead it gets cut off by the bottom of the viewport:

![image](https://user-images.githubusercontent.com/8772661/88806454-59d40700-d1a8-11ea-9b3b-50241e1f2a9b.png)

**Comment by KittenCodes** [comment]
@shoelaced Here's what I see on the Sandbox site: https://www.loom.com/share/8ba8c42f5fe5434b9a611aca228551f9.

Are there any plugins on your site that could be preventing the scroll?

**Comment by shoelaced** [comment]
I don't think you're on the same page that I am. Here's the sandbox homepage where I've set up the issue: https://oxygen-qm45fwrcr799k.oxygen-demo.qsandbox.me/

**Comment by KittenCodes** [comment]
@shoelaced You're right, I was on a different page. Sorry about that! I've reported the issue internally.
